specification/purview/Azure.Analytics.Purview.DataMap/client.tsp

```diff
@@ -0,0 +1,153 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using PurviewDataMap;
+
+@TypeSpec.Versioning.useDependency(PurviewDataMap.ApiVersions.v2023_09_01)
+namespace Customizations;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@client({
+  name: "EntityClient",
+  service: PurviewDataMap,
+})
+interface EntityClient {
```

Comment:
For python, operation group is better than sub client. Here is old package API: https://github.com/Azure/azure-sdk-for-python/blob/cb97ec0a23ebd59eb8fd6d5c338efc25026490ee/sdk/purview/azure-purview-catalog/azure/purview/catalog/_client.py#L71-L91